components/post_view/post_message_preview/index.ts

```diff
@@ -30,8 +34,10 @@ type Props = {
 
 function mapStateToProps(state: GlobalState, ownProps: Props) {
```

Comment:
Turn this into a makeMapStateToProps and move `makeGetChannel`. See here for an example, /components/threading/global_threads/thread_item/index.ts

Reply:
Thanks, I've made the changes 👍